WireDomain/Sources/WireDomain/Repositories/UserClients/UserClientsRepository.swift

```diff
@@ -190,7 +203,7 @@ public struct ClientRepository: ClientRepositoryProtocol {
             in: context
         ) else {
             return WireLogger.userClient.error(
-                "Failed to find existing client with id: \(id)"
+                "Failed to find existing client with id: \(id.obfuscated())"
```

Comment:
same here

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/2019/commits/f606230d354cb05894ca1ce4dc3fa8c1f3d0f244